pkg/sdk/privileges.go

```diff
@@ -139,22 +139,29 @@ const (
 	SchemaPrivilegeCreateProjectionPolicy            SchemaPrivilege = "CREATE PROJECTION POLICY"
 	SchemaPrivilegeCreateRowAccessPolicy             SchemaPrivilege = "CREATE ROW ACCESS POLICY"
 	SchemaPrivilegeCreateSessionPolicy               SchemaPrivilege = "CREATE SESSION POLICY"
-	SchemaPrivilegeCreateService                     SchemaPrivilege = "CREATE SERVICE"
 	SchemaPrivilegeCreateSecret                      SchemaPrivilege = "CREATE SECRET"
 	SchemaPrivilegeCreateSequence                    SchemaPrivilege = "CREATE SEQUENCE"
+	SchemaPrivilegeCreateService                     SchemaPrivilege = "CREATE SERVICE"
+	SchemaPrivilegeCreateSnapshot                    SchemaPrivilege = "CREATE SNAPSHOT"
 	SchemaPrivilegeCreateStage                       SchemaPrivilege = "CREATE STAGE"
 	SchemaPrivilegeCreateStream                      SchemaPrivilege = "CREATE STREAM"
 	SchemaPrivilegeCreateStreamlit                   SchemaPrivilege = "CREATE STREAMLIT"
 	SchemaPrivilegeCreateSnowflakeCoreBudget         SchemaPrivilege = "CREATE SNOWFLAKE.CORE.BUDGET"
 	SchemaPrivilegeCreateSnowflakeMlAnomalyDetection SchemaPrivilege = "CREATE SNOWFLAKE.ML.ANOMALY_DETECTION"
 	SchemaPrivilegeCreateSnowflakeMlForecast         SchemaPrivilege = "CREATE SNOWFLAKE.ML.FORECAST"
-	SchemaPrivilegeCreateTag                         SchemaPrivilege = "CREATE TAG"
+	SchemaPrivilegeCreateDynamicTable                SchemaPrivilege = "CREATE DYNAMIC TABLE"
+	SchemaPrivilegeCreateExternalTable               SchemaPrivilege = "CREATE EXTERNAL TABLE"
+	SchemaPrivilegeCreateHybridTable                 SchemaPrivilege = "CREATE HYBRID TABLE"
+	SchemaPrivilegeCreateIcebergTable                SchemaPrivilege = "CREATE ICEBERG TABLE"
 	SchemaPrivilegeCreateTable                       SchemaPrivilege = "CREATE TABLE"
+	SchemaPrivilegeCreateTag                         SchemaPrivilege = "CREATE TAG"
 	SchemaPrivilegeCreateTask                        SchemaPrivilege = "CREATE TASK"
+	SchemaPrivilegeCreateMaterializedView            SchemaPrivilege = "CREATE MATERIALIZED VIEW"
 	SchemaPrivilegeCreateView                        SchemaPrivilege = "CREATE VIEW"
 	SchemaPrivilegeModify                            SchemaPrivilege = "MODIFY"
 	SchemaPrivilegeMonitor                           SchemaPrivilege = "MONITOR"
 	SchemaPrivilegeUsage                             SchemaPrivilege = "USAGE"
+	SchemaPrivilegeCreateNotebook                    SchemaPrivilege = "CREATE NOTEBOOK"
```

Comment:
Nit: wrong order, should be on L132

Reply:
The order was determined by the documentation and CREATE NOTEBOOK is not yet there. That's why it landed at the bottom